packages/calcite-components/src/components/shell/shell.e2e.ts

```diff
@@ -122,3 +122,41 @@ describe("calcite-shell", () => {
     expect(panelTop).toBeNull();
   });
 });
+
+describe("slotted overrides", () => {
```

Comment:
Can you also add a default theme test (for `background`/`border-color` props) and one to cover the deprecated `--calcite-shell-tip-spacing`?